website/docs/reference/specifying-dependencies.md

```diff
@@ -1,6 +1,27 @@
 # Specifying Dependencies
 
-Your packages can depend on other libraries from Git repositories or subdirectories on your local file system.
+Your packages can depend on other libraries from registries, Git repositories, or subdirectories on your local file system.
+
+## Specifying dependencies from official registry
```

Comment:
As a reference, this should not be limited to the official registry, but also include a example of using another registry by url.

Reply:
Removed `official` from the header, and added an example of using custom registry i newest commit